bundle/Resources/public/admin/js/field/jquery.eztags.js

```diff
@@ -315,7 +315,6 @@
 
   Base.prototype.render_skeleton = function() {
     var $markup = $(this.render_template('skeleton'));
-    !this.opts.hasAddAccess && $markup.find('.button-add-tag').remove();
```

Comment:
Unfortunatelly, you cannot do this. This code is used interchangeably with other admin interfaces so this is a break in regards to them, especially since this is a Platform UI specific issue.

Reply:
@emodric Is there any chance to hide this button instead of removing it? Otherwise, I'll have to get html of this button and inject it once again after fetching `ez:tags:add` flag.

Reply:
Yep, that could work :+1: